screens/CameraRecording.tsx

```diff
@@ -28,24 +28,38 @@ const CameraRecording: FunctionComponent = () => {
             <View style={Styles.bottomRow}>
                 <Button style={Styles.flip} onPress={() => {
                     setCameraType(cameraType == Camera.Constants.Type.front ? Camera.Constants.Type.back : Camera.Constants.Type.front)
-                }}> 
-                    <Text style={Styles.text}>Flip</Text> 
+                }}>
+                    <Text style={Styles.text}>Flip</Text>
                 </Button>
                 <Button style={Styles.takePicture} onPress={() => {
                     if (camera) {
-                        camera.takePictureAsync()
-                            .then(pic => {
-                                console.log(pic.uri);
+                        camera.takePictureAsync({quality: 0.3})
+                            .then(async pic => {
+                                 fs.uploadAsync('https://content.dropboxapi.com/2/files/upload', pic.uri, {
```

Comment:
thoughts on pulling out this function into a separate file for API-call type stuff? so then here, it's just calling a function wihtout having to worry about the http headers and such